experimental/seaweedfs/istio/istio-authorization-policy.yaml

```diff
@@ -12,11 +12,23 @@ spec:
     - source:
         principals:
         - cluster.local/ns/kubeflow/sa/ml-pipeline
-  - from:
-    - source:
-        principals:
         - cluster.local/ns/kubeflow/sa/ml-pipeline-ui
-  # Allow traffic from User Pipeline Pods, which don't have a sidecar.
+    to:
+    - operation:
+        ports: ["8333"]
+  # Allow traffic to s3 endpoint from User Pipeline Pods, which don't have a sidecar.
+  - {}
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: pipeline-profile-controller
+spec:
+  action: ALLOW
+  selector:
+    matchLabels:
+      app: kubeflow-pipelines-profile-controller
+  rules:
   - {}
```

Comment:
What is this policy for ? Who should have access to pipelines-profile-controller? I think only metacontoller should call it. Please also compare with https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/installs/multi-user/istio-authorization-config.yaml and https://github.com/kubeflow/pipelines/blob/8d0ae5381e8366905c90009c56fd0e4807e94f0f/manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/deployment.yaml#L10